### PR TITLE
Tag Actors v0.1.0 [https://github.com/daqcore/Actors.jl]

### PR DESCRIPTION
**Pull request by oschulz**

### CODE REVIEW

File: Actors/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.5-
+Compat 0.20.0
```
Comment (tkelman) [inline Actors/versions/0.0.2/requires]: abstract type only requires 0.17

File: Actors/versions/0.0.2/requires [hunk 2]
```diff
@@ -0,0 +1,2 @@
+julia 0.5-
```
Comment (tkelman) [inline Actors/versions/0.0.2/requires]: drop the trailing dash if you don't need to support prereleases of 0.5

**Comment by oschulz** [comment]
Thanks for the hints, Tony.

**Comment by oschulz** [comment]
> missing an important line from https://github.com/daqcore/Actors.jl/pull/2/files

Sorry, @tkelman , no idea how that got lost in the merge. I'll fix it.

**Pull request merged**